src/com/projectkorra/projectkorra/BendingPlayer.java

```diff
@@ -639,7 +639,39 @@ public boolean isPermaRemoved() {
 	public boolean isToggled() {
 		return this.toggled;
 	}
-
+	/**
+	 * Checks if the {@link BendingPlayer} can bend any element
+	 * @return true if the player can bend an element
+	 */
+	public static boolean isBender(BendingPlayer bPlayer) {
+		try {
+		if(bPlayer.hasElement(Element.FIRE)) {

```

Comment:
Like the idea but I feel the execution could be a lot more elegant. I think a much better way to accomplish this would be to just check if the `BendingPlayer.elements` ArrayList is empty. That would save a lot of code and would account for custom elements.